website/conferences/model.py

```diff
@@ -25,6 +25,19 @@ class Conference(StoredObject):
     public_projects = fields.BooleanField(required=False, default=True)
     poster = fields.BooleanField(default=True)
     talk = fields.BooleanField(default=True)
+    field_names = fields.DictionaryField(
```

Comment:
Since we are introducing a new field, it seems that this change will require updating the existing database population script. (so that new conferences can be defined correctly from the start) Is that correct?

https://github.com/abought/osf.io/blob/develop/scripts/populate_conferences.py